### PR TITLE
docs - add docs for v5-to-6 migration and upgrade

### PR DESCRIPTION
**Pull request by lisakowen**
add some more docs for pxf version 5 to 6 upgrade.  (remember we have been collectively adding to this topic in other PRs.) this PR includes:
- HBase migration info
- description of pxf [cluster] migrate command
- upgrade procedure additions, including running the migrate command, dealing with HBase JAR, and dropping/recreating the extension

please double-check that the steps are in the correct order.

not in this PR:  pxf upgrade from the perspective of greenplum (i.e. the pxf upgrade/migration docs in the greenplum repo)

doc review site link:  http://docs-lisa-pxf6-migupd.cfapps.io/pxf/6-0/using/upgrade_5_to_6.html

### CODE REVIEW

File: docs/content/ref/pxf-cluster.html.md.erb [hunk 1]
```diff
@@ -47,6 +49,9 @@ The `pxf cluster` utility command manages PXF on the master, standby master, and
 <dt>init (deprecated)</dt>
 <dd>The command is equivalent to the `register` command.</dd>
 
+<dt>migrate</dt>
+<dd>Migrate the configuration in a PXF 5 `$PXF_CONF` directory to `$PXF_BASE` on each Greenplum Database host. When you run the command, you must identify the PXF 5 configuration directory via an environment variable named `PXF_CONF`. PXF migrates the version 5 configuration to `$PXF_BASE`, copying and merging files and directories as necessary. <div class="note">You must manually migrate any <code>pxf-log4j.properties</code> customizations to the <code>pxf-log4j2.xml</code> file.</div></dd>
```
[the remark this answers is not in the document]
Comment (lisakowen) [reply docs/content/ref/pxf-cluster.html.md.erb:53]: great suggestion!  i added an xref on the upgrade page - the user should first encounter the need to manually migrate the logging config there.

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 2]
```diff
@@ -46,9 +46,24 @@ After you install the new version of PXF, perform the following procedure:
     $ ssh gpadmin@<gpmaster>
     ```
 
-1. other steps here
+1. You must run the `pxf` commands specified in subsequent steps using the binaries from your PXF 6.x installation. Ensure that the PXF 6.x installation `bin/` directory is in your `$PATH`, or provide the full path to the `pxf` command.
```
[the remark this answers is not in the document]
Comment (lisakowen) [reply docs/content/upgrade_5_to_6.html.md.erb]: added the version command.  thanks!

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 3]
```diff
@@ -104,28 +119,42 @@ After you install the new version of PXF, perform the following procedure:
...
-        | -------------------|-------------|
-        | Hadoop profile names | hdfs:\<profile> as noted [here](access_hdfs.html#hadoop_connectors) |
-        | `jdbc.user.impersonation` property | `pxf.service.user.impersonation` property in the [jdbc&#8209;site.xml](jdbc_cfg.html#jdbcimpers) server configuration file |
-        | `PXF_KEYTAB` configuration property | `pxf.service.kerberos.keytab` property in the [pxf&#8209;site.xml](cfg_server.html#pxf-site) server configuration file |
-        | `PXF_PRINCIPAL` configuration property | `pxf.service.kerberos.principal` property in the [pxf&#8209;site.xml](cfg_server.html#pxf-site) server configuration file |
-        | `PXF_USER_IMPERSONATION` configuration property | `pxf.service.user.impersonation` property in the [pxf&#8209;site.xml](cfg_server.html#pxf-site) server configuration file |
+1. PXF 6.x includes a new version of the `pxf` extension. You must drop and re-create the extension in every Greenplum database in which you are using PXF. A database superuser or the database owner must run these SQL commands in the `psql` subsystem or in an SQL script:
+
+    ``` sql
+    DROP EXTENSION pxf;
```
[the remark this answers is not in the document]
Comment (frankgh) [reply docs/content/upgrade_5_to_6.html.md.erb:135]: Bradford is correct, we don't need dropping the extension to register PXF. However, we no longer copy the `.so` and `.sql` files to `$GPHOME`. We only copy the `.control` file when registering the extension. Maybe we can ask users to cleanup the old `pxf.so` and `.sql` files.

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 4]
```diff
@@ -2,15 +2,15 @@
...
 ---
 
-If you have installed, configured, and are using PXF in your Greenplum Database 5 or 6 cluster, you must perform some upgrade actions when you install PXF 6.x.
+If you have installed, configured, and are using PXF 5 in your Greenplum Database 5 or 6 cluster, you must perform some upgrade actions when you install PXF 6.x.
 
-<div class="note">If you are using PXF with Greenplum Database 5, you must upgrade Greenplum to version 5.21.2 or newer before you upgrade to PXF 6.</div>
+<div class="note">If you are using PXF with Greenplum Database 5, you must upgrade Greenplum to version 5.21.2 or newer before you upgrade to PXF 6.x.</div>
 
-The PXF upgrade procedure has two parts. You perform one procedure before, and one procedure after, you install a new version to upgrade to PXF 6:
+The PXF upgrade procedure has two parts. You perform one procedure before, and one procedure after, you install a new version to upgrade to PXF 6.x:
```
Comment (ashuka24) [inline docs/content/upgrade_5_to_6.html.md.erb:9]: This seems a bit confusing to me, could we rephrase it to something like: 
```suggestion
The PXF upgrade procedure has three main steps. You perform one pre-install procedure, the install itself, and then a post-install procedure in order to upgrade to PXF 6.x:
```

**Comment by lisakowen** [comment]
i believe that i have now made all of the requested updates to this PR.

**Review by denalex — APPROVED** [review]

**Comment by lisakowen** [comment]
merging, will address any further comments in a new PR.

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 5]
```diff
@@ -29,14 +29,14 @@ Perform this procedure before you upgrade to a new version of PXF:
     gpadmin@gpmaster$ pxf version
     ```
 
-1. add other steps here - TBD like backing up $PXF_HOME, $PXF_CONF
+1. Identify the file system location of the `$PXF_CONF` setting in your PXF 5.x PXF installation; you will need this later.
```
Comment (dyozie) [inline docs/content/upgrade_5_to_6.html.md.erb:32]: PXF 5.x PXF -> PXF 5.x
